src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleCloudStorageDAO.scala

```diff
@@ -22,6 +36,10 @@ class HttpGoogleCloudStorageDAO(clientSecretsJson: String, dataStoreFactory: Dat
   val gcsFullControl = "https://www.googleapis.com/auth/devstorage.full_control"
   val computeFullControl = "https://www.googleapis.com/auth/compute"
   val scopes = Seq(gcsFullControl,computeFullControl)
+  val directoryScopes = Seq(DirectoryScopes.ADMIN_DIRECTORY_GROUP)
+
+  val conf = ConfigFactory.parseFile(new File("/etc/rawls.conf"))
+  val gcsConfig = conf.getConfig("gcs")
```

Comment:
We should pass parameters into the DAOs instead of loading the conf file directly.